.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,36 @@
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Get version and SHA
+        id: version-sha
+        run: |
+          echo "##[set-output name=version;]$(cat VERSION)"
+          echo "##[set-output name=sha_id;]$(git rev-parse --short HEAD)"
+      - name: Docker login
+        uses: azure/docker-login@v1
+        with:
+          login-server: quay.io
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Docker build, tag, and push
+        run: |
+          echo "Building version ${IMAGE_VERSION} of image ${IMAGE_NAME}"
```

Comment:
for later, we should benefits from the existing cache

Improvement 1:
``
docker pull
docker build --cache-from
``

Improvement2
Move to buildx and probably introduce a PLATFORMS file where we have platforms for which we can build images
and we can merge both improvements as buildx handle that automatically for us.

Reply:
Noted :+1: